src/ui/ViewModels/FromUcasViewModel.cs

```diff
@@ -11,6 +11,8 @@ public class FromUcasViewModel
         public string CourseTitle { get; set; }
         public string AccreditingProviderId { get; set; }
         public CourseVariantViewModel Course { get; set; }
+        public CourseEnrichmentViewModel CourseEnrichment { get; set; }
```

Comment:
Why is there enrichment in a FromUcas model?
Does the model need renaming?

Minor.

Reply:
Yeah the FromUcas needs a rename, reason why the enrichment is there is simply cause FromUcas is the varient page view model